projects/vdk-heartbeat/src/vdk/internal/heartbeat/config.py

```diff
@@ -109,6 +109,12 @@ def __init__(self, config_file=None):
             "CONTROL_API_URL", "VDK_HEARTBEAT_CONTROL_SERVICE_URL", is_required=False
         )
 
+        # The VDK Frontend URL
+        # If no value is found, argument will not be passed and default will be used
+        self.frontend_url = self._get_atleast_one_value(
+            "FRONTEND_URL", "VDK_HEARTBEAT_FRONTEND_URL", is_required=False
```

Comment:
@murphp15 

>I would think that you shouldn't need the frontend_url. it will always be the control plane url too.
>The control plane will be hit by including /data-jobs in the url.


We have this in the post deploy test config.

```
CONTROL_API_URL=http://cicd-control-service-svc:8092
```

Not sure how it will work exactly with the port as part of the URL. I'd like to avoid parsing it if possible.